tests/dummy/app/templates/components/x-sidenav.hbs

```diff
@@ -0,0 +1,15 @@
+<section class="c-sidenav__header-section u-relative">
+  <header class="c-sidenav__header o-content-box u-fill">
+    {{#link-to 'index' class="c-sidenav__logo"}}Ember Data Twitch{{/link-to}}
```

Comment:
minor comment, but I'd prefer to standardize on using single quotes inside of htmlbars brackets `{{}}`, and then double-quotes everywhere else in hbs files.